includes/Settings.php

```diff
@@ -409,15 +409,34 @@ public function get_setting( string $key, $default = false ) {
 			 *
 			 * @var string|array<int|string,mixed>|bool
 			 */
-			return get_option( $key, $default );
+			$option = get_option( $key, $default );
+			if ( $option === $default ) {
+				return $option;
+			}
+		} else {
+			/**
+			 * Setting value.
+			 *
+			 * @var string|array<int|string,mixed>|bool
+			 */
+			$option = get_option( $key );
+		}
+
+		$settings = get_registered_settings();
+		if ( isset( $settings[ $key ] ) ) {
+			$value = rest_sanitize_value_from_schema( $option, $settings[ $key ] );
+			if ( is_wp_error( $value ) ) {
+				return $option;
```

Comment:
If not valid, should we return the default value?

Reply:
Sounds reasonable, just to be on the safe side.

Looks like the function only ever returns errors when using `anyOf` or `uniqueItems`, which we both don't use anyway.